src/main/webapp/WEB-INF/tags/instructor/course/activeCoursesTable.tag

```diff
@@ -12,6 +12,9 @@
       <th id="button_sortcoursename" class="button-sort-none toggle-sort">
         Course Name<span class="icon-sort unsorted"></span>
       </th>
+      <th id="button_sortcoursecreateddate" data-toggle-sort-comparator="sortDate" data-toggle-sort-extractor="dateStampExtractor" class="button-sort-none toggle-sort">
+        Course Created At<span class="icon-sort unsorted"></span>
```

Comment:
Use `Creation Date` for consistency with the sessions table. The word `Course` can be dropped for brevity.